indy-besu/smart_contracts/contracts/did/DidTypes.sol

```diff
@@ -13,6 +13,7 @@ struct DidDocumentStorage {
  * @dev VerificationRelationship links a DID to a verification method
  */
 struct DidMetadata {
+    address creator;
```

Comment:
Can we call it the same with DIDRegistry.ts ?

Reply:
Renamed